### PR TITLE
Add some file

### PR DESCRIPTION
**Pull request by rbd3**
adding readme.md, gitignore, index.html, style.css.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,77 @@
-# hello-microverses
\ No newline at end of file
+# hello-microverses
+
+# Hello-Microverse
```
Comment (shella12) [inline README.md]: - [ ] Please remove duplicate headings from your README as this makes your README look cluttered

File: README.md [hunk 2]
```diff
@@ -1 +1,77 @@
...
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
```
Comment (shella12) [inline README.md]: - [ ] I can see that you are using Microverse README template to document your project in your README. But there are some sections missing. Please refer to [this template](https://github.com/microverseinc/readme-template) and make sure all of the main sections are added to your README. The only section that is OPTIONAL is FAQ. You can always skip it if you want to. Remember to customize the template to your project as well.

File: linters.yml
Comment (shella12) [inline linters.yml:1]: - [ ] Please make sure to remove duplicate `linters.yml` file from project directory. `linters.yml` is only suppose to be copied into `.github/workflows` folder.  

Screenshot 2023-03-21 at 10 44 47 PM

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @rbd3 , 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Correct use of git flow ✅ 
- All required files are added

## Required Changes ♻️

- [ ] Great job on setting linter files however please make sure there aren't any linter errors reported and all linter checks on the github are green. AKA: Linter should report NO errors.


Screenshot 2023-03-21 at 10 46 29 PM

_Check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL] Please use descriptive commit messages. The commit message should be written in the imperative tense (which you have done 👍 ) and capitalized e.g. instead of` fix stylint error` say  ` Fix stylint error`. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). Its one of the requirements of the project but please try to keep this in mind for future

Screenshot 2023-03-21 at 10 47 17 PM

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (rbd3) [reply README.md]: @shella12 
i removed this duplicated headings.

File: README.md (hunk 2 above)
Comment (rbd3) [reply README.md]: @shella12 
i made some additions in my README

File: linters.yml
Comment (rbd3) [reply linters.yml:1]: @shella12 
i did it

File: README.md [hunk 3]
```diff
@@ -1 +1,160 @@
...
+- **[key_feature_2]**
+- **[key_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (braincee) [inline README.md]: - [ ] [OPTIONAL] Please remove the live demo link since you have not deployed your project.

File: README.md [hunk 4]
```diff
@@ -1 +1,160 @@
...
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/rbd3)
+- Twitter: [@twitterhandle](https://twitter.com/@Narson321)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/andry-narson-rabedesana-15b8b4248)
```
Comment (braincee) [inline README.md]: - [ ] [OPTIONAL] Please add your username of social media links inside [] brackets

**Review by braincee — APPROVED** [review]
Hi @rbd3 , I am @braincee, your Code Reviewer.

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

youre-approved-shirley-auto-group

OPTIONAL CHANGES
---

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @braincee._

---

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**